docker/run.sh

```diff
@@ -87,17 +105,31 @@ PGPASSWORD=$DB_PASSWORD psql "sslmode=verify-ca sslrootcert=/root/.postgresql/ro
 -c "\copy (SELECT row_to_json(t) FROM (SELECT * FROM casev2.cases where last_updated >= '$START_OF_PERIOD' and last_updated < '$END_OF_PERIOD') t) To '$CASES_FILE';"
 
 
-echo "zipping cases file"
+if [ -n "$DATAEXPORT_MI_BUCKET_NAME" ]
+then
+      echo "$DATAEXPORT_MI_BUCKET_NAME is empty"
+else
+    echo "adding cases file $CASES_FILE to bucket $DATAEXPORT_MI_BUCKET_NAME"
+    gsutil -q cp "$CASES_FILE" gs://"$DATAEXPORT_MI_BUCKET_NAME"
+fi
+
 
-filename=CensusResponseManagement_case_$PERIOD_DATE.zip
-zip "$filename" "$CASES_FILE"
+if [ -n "$DATAEXPORT_BUCKET_NAME" ]
+then
+      echo "$DATAEXPORT_BUCKET_NAME is empty"
```

Comment:
`-z` means "if env var is empty". `-n` means "if the env var is NOT empty". Your logic says "If the env var is not empty act like it _is_ empty".